### PR TITLE
Add new topics (conditionally change a band's visibility and reorder table cells)

### PR DESCRIPTION
**Pull request by VeraGorbyliova**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+The report created in this tutorial contains two Detail **sub-bands** with different report controls. These sub-bands are used to display discontinued and current products. 
+
+![1](../../../../../images/conditionally-change-a-bands-visibility.png)
+
+The steps below demonstrate how to change a band's visibility based on a field's value.
+
+1. Create formatting rule(s). 
+
+	- Select a report and click its smart tag. In the invoked actions list, click the ellipsis button for the **FormattingRules** property.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]:
```suggestion
	- Select a report and click its smart tag. In the invoked actions list, click the **FormattingRules** property's ellipsis button.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	
+		![how-to-conditionally-change-band-visibility-1](../../../../../images/how-to-conditionally-change-band-visibility-124975.png)
+	
+	- In the invoked **Formatting Rules Editor**, click the **Edit Rule Sheet...** button. The button invokes the **Formatting Rule Sheet Editor**. Click ![Buttons_Plus](../../../../../images/buttons_plus8644.png) to create a new formatting rule.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]:
```suggestion
	- In the **Formatting Rules Editor**, click the **Edit Rule Sheet...** button to invoke the **Formatting Rule Sheet Editor**. Click ![Buttons_Plus](../../../../../images/buttons_plus8644.png) to create a new formatting rule.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	
+		![](../../../../../images/eurd-web-conditionally-change-a-bands-visibility-add-new-rules.png)
+		
+		Here, two rules are added.
+
+  	- Specify a condition for the rule. Click the ellipsis button for the rule's **Condition** property. In the invoked **Formatting Rules Editor**, specify the required expression.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]:
```suggestion
  	- Specify a condition for the rule. Click the ellipsis button for the rule's **Condition** property and specify the expression in the invoked **Formatting Rules Editor**.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+		![How to - ConditionallySuppressControls_2](../../../../../images/howto_conditionallysuppresscontrols_28647.png)
+
+	  Here, the **[Discontinued] == true** expression is set for the **formattingRule1**  and the **[Discontinued] == false** expression - for the **formattingRule2**.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md:32]:
```suggestion
	  Here, the **formattingRule1** is set to **[Discontinued] == true**  and the **formattingRule2** is set to **[Discontinued] == false**. 
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+	- Specify the **Visible** property for the rule.
+
+        ![](../../../../../images/eurd-web-conditionally-change-a-bands-visibility-set-visibility.png)  
+
+        Here, the **Visible** property is set to **False** for both rules.
+
+	
+2. Apply the rule(s) to the required band(s).
+ 	
+	Invoke the **Formatting Rules Editor** for a selected a band and move a rule to the **Rules applied to a control** section.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]:
```suggestion
	Select a band, invoke the **Formatting Rules Editor**, and move a rule to the **Rules applied to a control** section.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+	
+	![](../../../../../images/change-a-bands-visibility-formatting-ruled-editor.png)
+	
+
+	Here, the **formattingRule1** is applied to the **SubBand1**, and the **formattingRule2** - to the **SubBand2**.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]:
```suggestion
	Here, the **formattingRule1** is applied to **SubBand1**, and the **formattingRule2** to **SubBand2**.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+---
+title: Conditionally Change a Band's Visibility
+owner: Vera Gorbyliova
+seealso: []
+---
+# Conditionally Change a Band's Visibility
+
+This topic describes how to change report band visibility.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md]:
```suggestion
This topic describes how to change the report band's visibility.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+Set a band's **Visible** property to an expression to conditionally change the band's visibility based on a field's value or a parameter.
+
+The report created in this tutorial contains two Detail **sub-bands** with different report controls. These sub-bands are used to display discontinued and current products.   
+
+![1](../../../../../images/conditionally-change-a-bands-visibility.png)
+
+The steps below demonstrate how to change a band's visibility based on a field's value.
+
+1. Select the required band and switch to the **Properties** panel. Choose **Expressions** ![](../../../../../images/conditionally-change-a-bands-visibility-expression-button.png) and click the **Visible** property's ellipsis button.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md]:
```suggestion
1. Select a band and switch to the **Properties** panel. Choose **Expressions** ![](../../../../../images/conditionally-change-a-bands-visibility-expression-button.png) and click the **Visible** property's ellipsis button.
```

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+	![](../../../../../images/conditionally-change-a-bands-visibility-visible-button.png)
+
+2. In the invoked **Expression Editor**, specify the required expression.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-expression-bindings/conditionally-change-a-bands-visibility-expression-bindings.md]:
```suggestion
2. In the invoked **Expression Editor**, specify the expression.
```

File: interface-elements-for-web/articles/report-designer/use-report-elements/use-tables/manipulate-table-elements.md [hunk 10]
```diff
@@ -38,6 +38,27 @@ You can resize columns equally in a similar way by selecting the columns or the
...
+
+    ![](../../../../images/eurd-web-table-green-light.png)  
+ 
+ - The red light indicates where the cell can't be dropped.
+
+    ![](../../../../images/eurd-web-table-cells-red-light.png) 
+
+
 ## Stretch Table Cells
 You can stretch a cell so that it occupies several rows and columns. To stretch a cell across several columns, remove a neighboring cell by pressing DELETE or selecting the **Delete Cell** command in the **Action** category.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/report-designer/use-report-elements/use-tables/manipulate-table-elements.md]:
```suggestion
You can stretch a cell to occupy several rows and columns. To stretch a cell across several columns, press DELETE or select the **Delete Cell** command in the **Action** category to remove a neighboring cell.
```

File: interface-elements-for-web/articles/report-designer/use-report-elements/use-tables/manipulate-table-elements.md [hunk 11]
```diff
@@ -38,6 +38,27 @@ You can resize columns equally in a similar way by selecting the columns or the
 
 ![](../../../../images/eurd-web-table-control-distribute-columns-evenly.png)
 
+## Reorder Table Cells
+
+In [Report Explorer](../../report-designer-tools/ui-panels/report-explorer.md) you can change the order of table cells. Select a cell and drag it to a new position.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/report-designer/use-report-elements/use-tables/manipulate-table-elements.md]:
```suggestion
You can change table cells' order in the [Report Explorer](../../report-designer-tools/ui-panels/report-explorer.md). To do this, select a cell and drag it to a new position.
```

**Review by dirk-pieterse — CHANGES_REQUESTED** [review]

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md (hunk 4 above)
Comment (VeraGorbyliova) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]: Why you use a dot after the word 'Here'? What do you mean?

Comment (VeraGorbyliova) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md]: You use dot after 'Here'. What do you mean?

Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md:32]: I accidentally changed the comma after 'Here' to a period.

Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/shape-report-data/shape-data-data-bindings/conditionally-change-a-bands-visibility-data-binding.md:32]: However, you use 'Here,' too many times. I suggest removing it or using an alternative.

**Review by dirk-pieterse — CHANGES_REQUESTED** [review]

**Pull request merged**